From: Dmitry Shachnev <mitya57@debian.org>
Date: Tue, 14 Sep 2021 19:34:48 +0300
Subject: Fix indentation error from sass

Error: Inconsistent indentation: 5 spaces were used for indentation,
but the rest of the document was indented using 2 spaces.
        on line 109 of src/sass/_theme_rst.sass
        from line 47 of src/sass/theme.sass

Forwarded: https://github.com/readthedocs/sphinx_rtd_theme/pull/1234
---
 src/sass/_theme_rst.sass | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/sass/_theme_rst.sass b/src/sass/_theme_rst.sass
index 372d318..c77ef72 100644
--- a/src/sass/_theme_rst.sass
+++ b/src/sass/_theme_rst.sass
@@ -106,11 +106,11 @@
       pointer-events: none
 
     span.linenos
-     display: inline-block
-     padding-left: 0px
-     padding-right: ($base-line-height / 2)
-     margin-right: ($base-line-height / 2)
-     border-right: 1px solid lighten($table-border-color, 2%)
+      display: inline-block
+      padding-left: 0px
+      padding-right: ($base-line-height / 2)
+      margin-right: ($base-line-height / 2)
+      border-right: 1px solid lighten($table-border-color, 2%)
 
   .code-block-caption
     font-style: italic
